### ISSUE TITLE
Determine source snapshots of merged snapshot via API

### ISSUE THREAD

**Issue by fpytloun** [aptly-dev/aptly#271]
This information is not available via API.
The only way how to determine source snapshots is to parse it's description, eg.:

```
source_snapshots = re.findall(r"'([\w\d-]+)'", snapshot['Description'])
```

But description format is not quaranteed.

**Comment by smira** [comment]
Yes, it's something missing right now. Internally each snapshot has UUID, and snapshot references its sources via UUIDs. So two things are missing:

* get snapshot info by UUID
* get source UUIDs

Internally it's a bit more complicated, as source for the snapshot could be mirror and repo as well.
